airbyte-integrations/connectors/source-tmdb/source_tmdb/tmdb.yaml

```diff
@@ -0,0 +1,294 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "https://api.themoviedb.org/3/"
+    http_method: "GET"
+    request_options_provider:
+      request_parameters:
+        api_key: "{{ config['api_key'] }}"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
```

Comment:
@btkcodedev you must implement Pagination here. you're only retrieving the first page data.
Check IncrementalPaginator: https://docs.airbyte.com/connector-development/config-based/understanding-the-yaml-file/pagination#page-increment

Reply:
Done